pyfranca/franca_parser.py

```diff
@@ -583,6 +585,67 @@ def p_map_def(p):
 
     # noinspection PyIncorrectDocstring
     @staticmethod
+    def p_constant_def(p):
+        """
+        constant_def : CONST type ID '=' INTEGER
+        """
+        if p[2].name not in [  "Int8",  "Int16",  "Int32",  "Int64",
```

Comment:
Such checks should be left to the processor as on the grammar level there is no way to deal with references.

Consider the following example:

```
typedef A is Int32
const A a = 42
```